docs/Troubleshooting.md

```diff
@@ -282,3 +283,63 @@ INGRESS_NGINX_CONFIG:
+* `V4_DEPLOYMENT_OPERATOR_ENABLED: true`, the [SAS Viya Platform Deployment Operator](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=itopscon&docsetTarget=p0839p972nrx25n1dq264egtgrcq.htm) will be installed into the cluster and used to deploy the SAS Viya platform
+* `V4_DEPLOYMENT_OPERATOR_ENABLED: false`, the [sas-orchestration command](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=itopscon&docsetTarget=p0839p972nrx25n1dq264egtgrcq.htm) whose tooling is delivered as a Docker image, is used to deploy the SAS Viya platform
+
+Alongside the two SAS Viya deployments methods, considerations for the two different ways that this project, viya4-deployment, can be run will also need to be made. You can either:
+* Clone this project and execute it [using the ansible-playbook](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/AnsibleUsage.md) binary you have installed on your host
+* Alternatively you can build a Docker image with the [Dockerfile](https://github.com/sassoftware/viya4-deployment/blob/main/Dockerfile) provided in this repository and run it using the [Docker run command](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/DockerUsage.md).
+
+The combination of setting `V4_DEPLOYMENT_OPERATOR_ENABLED: false` and running directly on your host using the `ansible-playbook` command is where using a provider based kubernetes configuration file is not compatible.
+
+When the `sas-orchestration` tooling is run (as a Docker container) to deploy SAS Viya into the cluster, the required binaries from the cloud provider for authentication are not present, meaning that the tooling will not be able to connect to the cluster to perform the deployment.
+
+When running the viya4-deployment project as a Docker container the `sas-orchestration` tooling is run in a slightly different manner to get around this limitation. We make use of `skopeo` to exact the contents of the `sas-orchestration` tooling image directly into our running viya4-deployment container. Since in our Dockerfile we include the installation of the required authentications binaries for GCP and AWS, the `sas-orchestration` tooling is able to make use of them and successfully connect to the kubernetes cluster.
+
+### Solution:
+
+You have a couple of options:
+* If you would still like to deploy the SAS Viya platform with the `sas-orchestration` command with your existing kubernetes configuration file, it is recommended to build Docker image for this project with the [Dockerfile](https://github.com/sassoftware/viya4-deployment/blob/main/Dockerfile) and run it using the [Docker run command](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/DockerUsage.md).
+* If you created your infrastructure with an Viya 4 IAC projects, you can go back and set `create_static_kubeconfig=true` and run `terraform apply` again to generate a "static" kubeconfig file that is compatible with `sas-orchestration`.
```

Comment:
Instead of "Viya 4 IAC projects" maybe "viya4-iac-* projects"? or just "Viya4 IAC projects".

Reply:
I think switching to viya4-iac-* is a good idea, makes it more clear that it's referring to the IAC project repos. Applying that suggestion